app/frontend/types.ts

```diff
@@ -256,3 +251,20 @@ export type Balance = {
   coins: Lovelace
   tokens: Token[]
 }
+
+export const enum AssetType {
+  ADA,
+  TOKEN,
```

Comment:
wouldn't it make sense to have type to describe the exact assetName and policyId of the given token? And have this enum instead as "AssetFamily" or "AssetKind" . The AssetType type then could be something like:

```
```
AssetType = {
    family: AssetFamily.ADA
} | {
    family: AssetFamily.TOKEN;
    policyIdHex: string,
    assetNameHex: string 
}
```
```